inventory/host_vars/local.timeoverflow.org/config.yml

```diff
@@ -1,6 +1,7 @@
 ---
 database_name: timeoverflow_development
 rails_environment: development
+database_user_role: SUPERUSER
```

Comment:
undefined in other stages?

Reply:
Will it :boom:? Should I add empty `database_user_role:` in other hosts?

Reply:
it will :boom: what are the default roles?